docs/tutorials/dockerimages.md

```diff
@@ -254,7 +254,7 @@ First, create a directory and a `Dockerfile`.
     $ touch Dockerfile
 
 If you are using Docker Machine on Windows, you may access your host
-directory by `cd` to `/c/Users/your_user_name`.
+directory by `cd` to `c:/Users/your_user_name`.
```

Comment:
yup - that's what I find - either proper windows `\` paths in the cmd.exe or powershell, or in git bash `/c/Users...`, or when using MobaXTerm its something like `/drives/C/Users..`

as this example is when using the toolbox based git bash quickstart, its not technically wrong - but in the context that many readers will be using Docker for windows/mac, we should add info for those too (so the difference i clearer.